tests/test_interfaces.py

```diff
@@ -0,0 +1,7 @@
+import unittest
+
+# from lsst.ts import tunablelaser
+
+
+class TestLaserInterface(unittest.IsolatedAsyncioTestCase):
+    pass
```

Comment:
Please open a Jira ticket for implementing this test class and refer to it here in a TODO comment.